FogCreek/Glitch-Community
# Wealthy daughter: best a11y practices for team analytics time filter

## Links
* Remix link: https://wealthy-daughter.glitch.me
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/144

## Changes:
* change Team Analytics time filter to work like a form element (select with arrow keys, not tab)

## How To Test:
* log into https://wealthy-daughter.glitch.me
* visit https://wealthy-daughter.glitch.me/@community
* the time filter (with the "Last 2 Weeks" button) should work like a `<select>` element

## Feedback I'm looking for:
Is this the correct behavior?
